core/templates/pages/story-editor-page/modal-templates/new-chapter-title-modal.component.ts

```diff
@@ -168,13 +168,6 @@ export class NewChapterTitleModalComponent implements OnInit {
     }
 
     this.invalidExpId = false;
-    const correctnessFeedbackIsEnabled = (
-      await this.curatedExplorationValidationService
-        .isCorrectnessFeedbackEnabled(this.explorationId));
-    if (!correctnessFeedbackIsEnabled) {
-      this.correctnessFeedbackDisabled = true;
-      return;
-    }
     this.correctnessFeedbackDisabled = false;
```

Comment:
Why do we need this variable? (Please also check if anything is referencing it, and handle that properly.)